src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultControllerArgumentBinder.cs

```diff
@@ -19,22 +17,16 @@ namespace Microsoft.AspNetCore.Mvc.Internal
     /// </summary>
     public class DefaultControllerArgumentBinder : IControllerArgumentBinder
```

Comment:
The action invoker should probably just DI the `ParameterBinder` and use a `for` loop.